secboot/encrypt_sb_test.go

```diff
@@ -53,6 +53,10 @@ func (s *encryptSuite) TestFormatEncryptedDevice(c *C) {
 			c.Assert(opts, DeepEquals, &sb.InitializeLUKS2ContainerOptions{
 				MetadataKiBSize:     2048,
 				KeyslotsAreaKiBSize: 2560,
+				KDFOptions: &sb.KDFOptions{
+					MemoryKiB: 32768, TargetDuration: 0,
+					ForceIterations: 4, Parallel: 0,
```

Comment:
from the cryptsetup docs is not super clear if Parallel must be set as well when setting ForceIterations or not. I suppose we need to test the effect of this

Reply:
It doesn't need to be set - it defaults to the maximum (4) if it's unset. In all cases, it's then adjusted down if there are fewer CPUs. The only requirement is that you can't try to set it to zero.

Reply:
Only for context:
which probably translates to 4 or 1 in real world I'd imagine. Apart of intel there are not many under 4 cores devices. On arm we either see 4+ or 1 in very low power devices.